src/views/NewProduct.vue

```diff
@@ -43,9 +43,9 @@ export default class NewProduct extends Vue {
 
   async removeExistingProduct(product: Product) {
```

Comment:
This function is not needed anymore, actually. `@remove`  at the top will never be emitted, so we can delete it.